server/src/database/migrations/20200503201112-add-lastUpdatedAchievementsAt-player.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+  up: (queryInterface, Sequelize) =>
+    queryInterface.sequelize.transaction(transaction =>
+      Promise.all([
+        queryInterface.addColumn(
+          'players',
+          'lastUpdatedAchievementsAt',
+          {
+            type: Sequelize.DATE
+          },
+          { transaction }
+        ),
+        queryInterface.bulkDelete('achievements', {}, { transaction })
```

Comment:
Deleting achievements and starting over seemed like a better way to go, will have to see how this meshes with your new achievements code

Reply:
I could remove this and do a check specifically for Date(0), but its hard to check if a date is before the previous one if its as far back as possible